### ISSUE TITLE
Kicking grilles breaks them faster than hitting them with objects

### ISSUE THREAD

**Issue by Iamgoofball** [NTStation/NTstation13#86]
_Issue by **[Tenebrosity](https://github.com/Tenebrosity)** from Monday Jun 17, 2013 at 10:21 GMT_
_Originally opened as https://github.com/tgstation/-tg-station/issues/796_

---
##### Server Revision:

_unknown_
##### Server:

Sibyl1 (17 June 2013)
###### What did you expect to happen?

I expected kicking a grille would break it slower than hitting it with a fire extinguisher, toolbox, wrench, etc.
###### What happened instead?

Kicking grilles is a far more effective way of breaking them than hitting them with a heavy object.
###### Why is this bad/What are the consequences?

Every crewmember is wearing steel-toe shoes and has calves of fury.
##### Steps to reproduce the problem:
1. Build two grilles
2. Count how many times it takes you to destroy the first one by clicking it with an empty hand
3. Count how many times it takes you to destroy the second one by clicking it with an object (wirecutters or a welder don't count)

Result: It's more effective to kick grilles than wail on them with heavy objects.
##### Anything else?

I know it's not a bug per se, but it's an annoying inconsistency.

**Comment by Iamgoofball** [comment]
_Comment by **[errorage](https://github.com/errorage)** from Monday Jun 17, 2013 at 13:56 GMT_

---

Grilles have 10HP, every time you kick it, you do 3HP of damage, 5HP of damage if you're a hulk or alien, rand(2,3) damage if you're a slime, melee_damage_upper if you're a simple animal

For items (amount of HP it reduces): 
Bullets: Proj.damage*0.2
Items that do fire damage: W.force
Items that do brute damage (or shards): W.force \* 0.1

What this means is that kicking a grille will do 3 HP of damage every time. Using a toolbox on a grille (which has force 5) will do 0.5HP damage per hit, using a lit welder on a grille will do 15 HP damage (because it does fire damage)

In other words, this issue is confirmed. Fixing it should be simple, please post opinions on how much damage each of these categories of attack (kicking, bulelts, items that do fire damage, items that do brute damage) should do.

**Comment by Iamgoofball** [comment]
_Comment by **[Tenebrosity](https://github.com/Tenebrosity)** from Monday Jun 17, 2013 at 19:08 GMT_

---

Currently working on some values for this.

**Comment by Iamgoofball** [comment]
_Comment by **[Tenebrosity](https://github.com/Tenebrosity)** from Monday Jun 17, 2013 at 19:26 GMT_

---

**Kicking:** 0.84 HP damage (12 kicks to break)
**Bullets:** Proj.damage \* 0.025 (40 shots with .38, beanbags; 7 shots with .357, lethals)
**Items that do fire damage:** W.force \* 0.5 (2 hits with a welder)
**Items that do brute damage:** W.force \* 0.25 (4 hits with a fire extinguisher, 8 hits with a toolbox)

What about lasers? They do 20 burn damage to humans, but currently grilles take 3 shots (60 damage) to break. You said "Items that do fire damage: W.force". If a grille's health is 10, surely one laser should break it currently? Or are fire damage and burn damage different?

**Comment by Iamgoofball** [comment]
_Comment by **[errorage](https://github.com/errorage)** from Monday Jun 17, 2013 at 19:37 GMT_

---

Lasers will probably follow bullet damage conditions.

**Comment by Iamgoofball** [comment]
_Comment by **[Giacom](https://github.com/Giacom)** from Monday Jun 17, 2013 at 19:37 GMT_

---

This isn't a bug.

**Comment by Iamgoofball** [comment]
_Comment by **[Giacom](https://github.com/Giacom)** from Monday Jun 17, 2013 at 19:46 GMT_

---

You can still change the kick damage in a pull request.

**Comment by Iamgoofball** [comment]
_Comment by **[errorage](https://github.com/errorage)** from Monday Jun 17, 2013 at 19:46 GMT_

---

I disagree. Also no matter if it's a bug or not, it's an issue.

**Comment by Iamgoofball** [comment]
_Comment by **[Giacom](https://github.com/Giacom)** from Monday Jun 17, 2013 at 19:47 GMT_

---

This has been considered a well known feature for the past few years; it isn't an "issue".

**Comment by Iamgoofball** [comment]
_Comment by **[EuroNumbers](https://github.com/EuroNumbers)** from Monday Jun 17, 2013 at 20:09 GMT_

---

This is really hardly an issue. Please keep RFE discussion limited here in issue tracker, it's not for feature discussion but for bugs. The forums are for that. If you want to change these values do a pull and have people vote on the changes instead of discussing them here.

This issue is closed. Don't reopen it, please.

**Comment by Iamgoofball** [comment]
_Comment by **[Tenebrosity](https://github.com/Tenebrosity)** from Monday Jun 17, 2013 at 20:36 GMT_

---

http://www.ss13.eu/phpbb/viewtopic.php?f=10&t=617&p=13751

**Comment by Iamgoofball** [comment]
_Comment by **[Nodrak](https://github.com/Nodrak)** from Tuesday Jun 18, 2013 at 00:16 GMT_

---

Numbers, a headcoder, has asked that this be closed. If you have a problem with his decision, you can always bring it up with the other headcoders or Numbers himself. I believe we're a reasonable lot... but please don't take it upon yourself to overrule the decision. 

Personally I believe this should be considered an issue, it seems to fit more as an 'issue' than a 'feature request'. Since no one has actually asked to have this re-opened, I have to keep it closed. Sorry Aran.

**Comment by Iamgoofball** [comment]
_Comment by **[Aranclanos](https://github.com/Aranclanos)** from Tuesday Jun 18, 2013 at 00:42 GMT_

---

pfff ahhahaa

**Comment by Iamgoofball** [comment]
_Comment by **[Giacom](https://github.com/Giacom)** from Tuesday Jun 18, 2013 at 12:40 GMT_

---

Aran you are being unreasonable.

**Comment by Iamgoofball** [comment]
_Comment by **[errorage](https://github.com/errorage)** from Tuesday Jun 18, 2013 at 15:10 GMT_

---

Will you please stop spamming my email already? Who cares what your personal definition of 'issue' is. The 'issue' of what the definition of 'issue' is, is irrelevant!

**Comment by Iamgoofball** [comment]
_Comment by **[Aranclanos](https://github.com/Aranclanos)** from Tuesday Jun 18, 2013 at 15:23 GMT_

---

get off of my lawn!

**Comment by Iamgoofball** [comment]
_Comment by **[Tenebrosity](https://github.com/Tenebrosity)** from Friday Jul 26, 2013 at 03:56 GMT_

---

The stupid thing is that I did as Errorage suggested and came up with some values that made sense to me. But rather than someone who can code just using those and tweaking them to their own preferences, you're going to play issue open and close fucking ping pong for the next 6 months.

And while I recognise that this isn't unintended functionality, it is a stupid thing about the game, and an exploit.
